Resources/Prototypes/Catalog/Fills/Items/misc.yml

```diff
@@ -38,3 +38,12 @@
       item:
       - KukriKnife
 
+- type: entity
+  id: ClothingShoesBootsSyndieFilled
+  parent: ClothingShoesBootsCombat
+  suffix: Filled
```

Comment:
```suggestion
  suffix: Filled, Throwing Knife
```
Also need similar suffix changes on default boots

Reply:
> Also need similar suffix changes on default boots

Do you want me to do this to every entity in ``misc.yml`` ?
Or uuuh on ``ClothingShoesBootsSecFilled`` ?

Reply:
No, only this type of boots